### PR TITLE
docs: recommend Sifive August 2020 toolchain

### PR DESCRIPTION
**Pull request by danc86**
Eventually we'll just use conda but in the meantime, update it.

### CODE REVIEW

**Review by alanvgreen — APPROVED** [review]

File: docs/source/setup-guide.rst [hunk 1]
```diff
@@ -35,19 +35,19 @@ Step 2: Install Vivado and Renode
 Step 3: Install RISCV toolchain
 -------------------------------
 
-1. Download the `April 2020`_ toolchain from freedom-tools and unpack the binaries to your home directory:
+1. Download the `August 2020`_ toolchain from freedom-tools and unpack the binaries to your home directory:
 
-.. _`April 2020`: https://github.com/sifive/freedom-tools/releases/tag/v2020.04.0-Toolchain.Only
+.. _`April 2020`: https://github.com/sifive/freedom-tools/releases/tag/v2020.08.0
```
Comment (alanvgreen) [inline docs/source/setup-guide.rst:40]: April -> August

Comment (danc86) [reply docs/source/setup-guide.rst:40]: Oops, I rushed this PR and it shows. Luckily we have CI to catch this mistake!

**Comment by danc86** [comment]
Well, the docs build fails but the error is:

```
Exception occurred:
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/docutils/writers/html5_polyglot/__init__.py", line 445, in section_title_tags
    if (ids and self.settings.section_self_link
AttributeError: 'Values' object has no attribute 'section_self_link'
```

which according to https://github.com/sphinx-doc/sphinx/issues/9825 seems to be because we are wrongly using docutils 0.18. So I guess it's probably broken for us on `main` too, not related to this PR.

Sphinx wants docutils < 0.18 because it seems docutils made some incompatible changes. However we have a somewhat older Sphinx and possibly it does not express the docutils < 0.18 version constraint so pip is giving us this incompatible mixture.

**Comment by danc86** [comment]
PR #340 should fix the doc build failure in CI.

**Comment by tcal-x** [comment]
Hi @danc86 , #340 is merged, so can you rebase this to see if it now passes?

**Comment by danc86** [comment]
Looks like this is all good now!

**Pull request merged**